crawler/src/main/java/de/unidisk/view/results/ProjectResultTable.java

```diff
@@ -56,6 +60,7 @@ public void load(String projectId){
                 projectStatusMessage = null;
             }
 
+            results = projectRepository.getResults(this.projectId);
```

Comment:
Hier übrprüfst du nicht, ob die projectId nicht auch leer ist (weiter oben Zeile "final Project p = projectRepository.getProject(projectId)" auch nicht). Kann aber passieren. Es führt meisten zu, Integer.parseInt() der eine NumberFormatException wirft. Probier das mal in den Funktionen abzufangen.